annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/field/FieldDeclarationStore.java

```diff
@@ -125,9 +126,11 @@ public ImmutableSet<String> getInLineMultipleFieldDeclarationsOnField(
   public OnField getLocationOnField(String clazz, String field) {
     FieldDeclarationInfo candidate =
         findNodeWithHashHint(node -> node.clazz.equals(clazz), FieldDeclarationInfo.hash(clazz));
+    Set<String> fieldNames = Sets.newHashSet(field);
     if (candidate == null) {
-      return null;
+      // field is on byte code.
+      return new OnField(clazz + ".java", clazz, fieldNames);
```

Comment:
If the fix is in bytecode, then it doesn't get applied right? Since `[clazz].java` won't be found as a full path. Maybe there needs to be a bit more of an explanation on the comment of what this dummy(?) value is for?

Reply:
51d9020c2373082f6856fe3cdaa8c4e71734074b Yes they will never get applied. But, I thought it is better to not create an instance of dummy locations to avoid complexities. Removed it 51d9020c2373082f6856fe3cdaa8c4e71734074b